static/app/components/feedback/list/useMailboxCounts.tsx

```diff
@@ -0,0 +1,59 @@
+import {useMemo} from 'react';
+
+import type {Organization} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+import useLocationQuery from 'sentry/utils/url/useLocationQuery';
+
+interface Props {
+  organization: Organization;
+}
+
+const MAILBOX = {
+  unresolved: 'is:unassigned is:unresolved',
```

Comment:
oo are we getting the count based on unassigned issues, rather than unread issues (that seems harder)?